### PR TITLE
[PATCH] Fixed issue when assignee is removed from an event.

### LINKED ISSUE #249: [Bug] Removing an assignee from an event is not working
### Steps to reproduce

1. Log in to an account
2. Go to the calendar endpoint
3. Create an event and assign a few people
4. Click on the event to update it
5. Remove an assignee
6. Click 'Update'
7. Click on the event again

### Expected results

When removing an assignee from an event, the user should be removed from the event.

### Actual results

The assignee is still appearing in the event assignee list.

### Screenshots

4.
image

5.
image

7.
image

### Relevant log output

```shell
No log output
```

### Severity

High

### PR DESCRIPTION
**Pull request by Manraj45**
closes #249

### CODE REVIEW

**Review by nasaku898 — APPROVED** [review]
Great work!

File: backend/src/main/routes/EventRoute.ts [hunk 1]
```diff
@@ -78,7 +78,6 @@ export default class EventRoute extends CommonRoutesConfig {
 
     this.getApp()
       .route(`/event/:id/accept`)
-      .all(checkJwt)
```
Comment (huangs08) [inline backend/src/main/routes/EventRoute.ts:81]: Why are we removing the token check for this endpoint call?

Comment (Manraj45) [reply backend/src/main/routes/EventRoute.ts:81]: We do not need to login when accepting an invite. The link is sent to the email. Its the same principal as when we accept the vaccination appointment with clicsante.

Comment (huangs08) [reply backend/src/main/routes/EventRoute.ts:81]: That makes sense. The rest looks good!

**Review by huangs08 — APPROVED** [review]

**Pull request merged**